tools/dockerfile/grpc_artifact_python_manylinux2014_x64/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Docker file for building gRPC manylinux Python artifacts.
+
+FROM quay.io/pypa/manylinux2014_x86_64
+
+# Update the package manager
+RUN yum update -y
+RUN yum install -y curl-devel expat-devel gettext-devel openssl-devel zlib-devel
+
+###################################
+# Install Python build requirements
+RUN /opt/python/cp35-cp35m/bin/pip install --upgrade cython
+RUN /opt/python/cp36-cp36m/bin/pip install --upgrade cython
+RUN /opt/python/cp37-cp37m/bin/pip install --upgrade cython
+RUN /opt/python/cp38-cp38/bin/pip install --upgrade cython
```

Comment:
Since this is a work trying out latest Python builds, can we also add Python 3.9 here? It might help us later, when we want to add 3.9 support.

Reply:
Sure. I added them along with the update-date comment so that we can upgrade docker images easily by modifying its date.